src/demo-app/drag-drop/drag-drop-demo.html

```diff
@@ -0,0 +1,38 @@
+<div class="list">
```

Comment:
Someone Andrew has been pushing for is to stop adding demos that only live in `demo-app/` and instead always embed something from the examples, since it forced us to flesh those out. Not necessary for this PR, but would be nice to do.